platform/security/encryption/platform-security-encryption-impl/src/test/java/ddf/security/encryption/impl/EncryptionServiceImplTest.java

```diff
@@ -69,6 +69,27 @@ public void testBadSetup() throws Exception {
     assertEquals(unWrappedDecryptedPassword, unencryptedPassword);
   }
 
+  @Test
```

Comment:
📘 This test tests:
1. If mutliple instances of this class will use the same `keyset`
2. If an instance of this class can decrypt a value that was encrypted by another instance of this class.